### PR TITLE
Team Page

### PR DESCRIPTION
**Pull request by arindom-s**
Team page implemented with necessary hover effects. Final changes to be done

### CODE REVIEW

**Comment by JDeepD** [comment]
Change base branch to `main` @arindom-s

File: src/app/team/page.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+"use client"
+
+import { Outfit } from 'next/font/google';
```
Comment (JDeepD) [inline src/app/team/page.tsx:3]: Remove this and use `font-outfit` utility.

File: src/components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+            <h1 style={customHoverDetails}
+              className="font-rp1 w-[400px] text-center mb-8 text-2xl mr-12">{designation}</h1>
+          </div>
+          <div className="fixed flex flex-col z-[5] right-12 gap-6" style={hover ? {
+            height: "max-content",
+            transform: "scale(1)",
+            transition: "all 0.3s ease",
+          } : { opacity: "0", transition: "all 0.3s ease", transform: "scale(0.6)" }}>
+            <a className="relative cursor-pointer" style={customHoverButton}>
+              <Image src="/team/fb.svg" height={30} width={30} alt="linkedin" />
```
Comment (JDeepD) [inline src/components/Card.tsx:59]: Use react icons.

File: src/components/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+            </a >
+            <a className="relative cursor-pointer" style={customHoverButton}>
```
Comment (JDeepD) [inline src/components/Card.tsx:61]: Show icon only when links present in data.

File: tailwind.config.ts [hunk 4]
```diff
@@ -33,8 +33,14 @@ const config = {
         rp1: ["readyplayer", 'monospace'],
         outfit: ['Outfit', 'sans-serif']
       },
+      backgroundImage:{
```
Comment (JDeepD) [inline tailwind.config.ts:36]: Fix build.

**Review by JDeepD — CHANGES_REQUESTED** [review]

File: src/app/team/page.tsx (hunk 1 above)
Comment (arindom-s) [reply src/app/team/page.tsx]: done

File: tailwind.config.ts (hunk 4 above)
Comment (arindom-s) [reply tailwind.config.ts]: done

File: src/app/team/page.tsx [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                </div>
+                <div className='flex justify-center mb-12 text-white text-xl gap-8'>
+                    <button style={team==1?{backgroundColor:"#59CAFA"}:{}} onClick={()=>{
+                        setTeam(1);
+                    }} className='group bg-[#fff1] flex items-center justify-between rounded-full px-[2vw] py-[0.75vw] shadow-[inset_1px_2px_2.5px_rgba(255,255,255,0.3),inset_1px_-2px_2.5px_rgba(255,255,255,0.3)] duration-1000 hover:shadow-[inset_1px_2px_2.5px_rgba(1,163,245,0.5),inset_1px_-2px_2.5px_rgba(1,163,245,0.5)]'>
+                        <p className="font-outfit group-hover:text-customBlue group-hover:scale-90 duration-300">Tech Team</p>
+                    </button>
+                    <button style={team==2?{backgroundColor:"#59CAFA"}:{}} onClick={()=>{
+                        setTeam(2);
+                    }} className='group bg-[#fff1] flex items-center justify-between rounded-full px-[2vw] py-[0.75vw] shadow-[inset_1px_2px_2.5px_rgba(255,255,255,0.3),inset_1px_-2px_2.5px_rgba(255,255,255,0.3)] duration-1000 hover:shadow-[inset_1px_2px_2.5px_rgba(1,163,245,0.5),inset_1px_-2px_2.5px_rgba(1,163,245,0.5)]'>
```
Comment (byt3h3ad) [inline src/app/team/page.tsx:25]: image

text invisible on hover

File: src/app/team/page.tsx [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+import Card, { MemberCard } from '~/components/Card';
+import data from "../../../public/team.json";
+import Marquee from 'react-fast-marquee';
+import { useState } from 'react';
+
+
+export default function Team() {
+    const [team, setTeam] = useState<1|2>(1);
+     return (
+        <div className='min-h-screen'>
```
Comment (byt3h3ad) [inline src/app/team/page.tsx:12]: image

content underflow in small screens

**Review by byt3h3ad — CHANGES_REQUESTED** [review]

**Pull request merged**